### DISCUSSION TITLE
Non-model genome annotation?

### DISCUSSION THREAD

**Discussion by benluttinen** [stuart-lab/signac#856] [General]
Hi everyone,
I'm currently working with 10X Multiome data for two non-model invertebrates. I've been trying to find a way to upload genome annotations to Signac but have been unsuccessful thus far. Has anyone had experience with this or knows of a work-around? Any help or suggestions would be greatly appreciated!

**Comment by timoast** [comment]
See https://satijalab.org/signac/articles/faq.html#an-annotation-or-genome-sequence-for-my-organism-is-not-available-on-bioconductor-what-do-i-do-
